### ISSUE TITLE
Is pyload download speed limited

### ISSUE THREAD

**Issue by cph001** [pyload/pyload#4439]
### Question

Hi,
I have a 2 gbit internet connexion and with pyload ng max peak is 1gbit.
With jdownloader I can max my connextion for download.
I can't figure out why pyload cannot use my max speed.

**Comment by vuolter** [comment]
No, pyload is not rate limited by default, have you tried scheduling multiple downloads at once? (any debug log would be helpful)

**Comment by cph001** [comment]
Yes in that case pyload use my max connection. But I was expecting for a single download file, it would also use my max connection

**Comment by GammaC0de** [comment]
You haven't mention which version of pyLoad do you use

**Comment by cph001** [comment]
sorry it's 0.5.0

**Comment by vuolter** [comment]
@cph001 some our hoster plugins allow only one chunk per file (aka one parallel connection for single download), so the download may be limited by the speed of the server itself; even that our plugins may have a chunk limit for various reasons (avoid ip/account banning, chunk data corruption, excessive slowdown of chunk merging, etc.).

@GammaC0de nice to hear from you again, I was a little worried, lol.

**Comment by cph001** [comment]
Yes I understand. I used realdebrid but thing is with jdownloader was able to use my max connection with the same debrid service with a single download.
